release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -126,6 +126,13 @@ For more information, see xref:../extensions/ce/managing-ce.adoc#olmv1-deploying
 
 Because {hcp} releases asynchronously from {product-title}, it has its own release notes. For more information, see xref:../hosted_control_planes/hosted-control-planes-release-notes.adoc#hosted-control-planes-release-notes[{hcp-capital} release notes].
 
+[id="ocp-release-notes-hcp-openstack-tp_{context}"]
```

Comment:
This is here because the feature will be working for OCP 4.19. The next HCP release is in July.